### DISCUSSION TITLE
In case of no grid power- will the battery still be charged from solar?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#953] [Q&A] [answered]
Genuine question, say there’s an outage and there’s no grid power but we have plenty of solar.
There’s a Victron MPPT, a battery with BMS and all connected to inverter via OpenDtu-onBattery.

Question: will the battery still be charged or when the inverter/DTU sees no grid it disables all the functionality?

Hope my question makes sense.

**Comment by genmad** [comment, marked as answer]
Yes, the victron will charge the battery.
Open Dtu may be off due to the power outage, this will not matter. The Hoymiles can‘t deliver any energy due to outage.

**Reply by greymda** [reply]
i was under the (possibly wrong?) impression that we still need opendtu-onbattery so the Victron would know the SoC of the battery, or i got the idea wrong?
because if no grid power, the esp32 module will be off hence no communication possible.

**Reply by spcqike** [reply]
No Victron charges the battery as stand alone. No need for openDTU to be around for that. 

OpenDTU on battery only manages the Hoymiles inverter. It can’t manage or stop the charging at all. That’s why it’s important to setup your solar charger properly (especially cutoff voltage)

**Reply by greymda** [reply]
@spcqike then why we connect the VE.direct cable from the Victron to OpenDtu?

**Reply by spcqike** [reply]
To read data from Victron VE.direct bus. E.g. live solar input power. So we can accomplish solar passthrough and thinks.

**Reply by greymda** [reply]
that makes sense, thanks! last question: when  you say “That’s why it’s important to setup your solar charger properly (especially cutoff voltage)” can you help with a screenshot please? i can’t find this on the Victron’s settings page.

**Reply by schlimmchen** [reply]
> i can’t find this on the Victron’s settings page.

Use the Victron smartphone app to configure your SmartSolar MPPT.

**Reply by greymda** [reply]
the only problem i have a BlueSolar so no bluetooth and hence no app. bit will figure it out somehow.

**Reply by schlimmchen** [reply]
I think you need this: https://www.victronenergy.com/communication-centres/ve-direct-bluetooth-smart-dongle

**Reply by greymda** [reply]
yes, i was looking at this, but then i’ll loose the connection via cable to opendtu, as the ve.direct port will be occupied by the bt dongle.

**Reply by DonJohnLong** [reply]
You can use the pre programmed lifepo4 profile, or you can disconnect the blutooth dongle after programming.

**Reply by greymda** [reply]
i’ll try saving up some money, will do a diy usb ve.direct cable as it seems that i need it for a couple of occasions.

**Reply by stegbth** [reply]
Hi @genmad ,

currently I'm a bit surprised, about this this answer, seems I haven't understood opendtu on battery completly ;)
According to https://www.victronenergy.com/live/battery_compatibility:pylontech_phantom?do=export_pdf 
The battery needs to communicate via canbus (or rs485) with the GX or Venus-Device, which controls on the other side the Victron MPPT.
This task I estimated would be taken by OpenDTU on Battery.
but
1. How does this work, when there is no transmit from opendtu to MPPT?
2. this would also answer the question why this would work without OpenDTU on battery.

br
Thomas

**Reply by genmad** [reply]
The mppt is a solar battery charger, therefore it handles the battery charging. It can charge non smart batteries as well as smart batteries.

smart batteries communicate back to the mppt and therefore it can charge them according to their wishes. Non smart batteries will be charged accordingly to the settings in the mppt, which are types of batterie and voltages. 

Open dtu, by just listening to the mppt, uses the „dumb“ approach for the batterie. Therefore it is important to set up the mppt correctly witch type and voltages.

I hope this answered your question @stegbth

**Comment by Manos1966** [comment]
> that makes sense, thanks! last question: when you say “That’s why it’s important to setup your solar charger properly (especially cutoff voltage)” can you help with a screenshot please? i can’t find this on the Victron’s settings page.

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Battery%E2%80%90Tipps#useful-comments-about-pylontech-batteries

**Reply by greymda** [reply]
i don’t have a pylontech, it’s a diy with jk-bms
